src/components/GridKeyboardNavigationContext/GridKeyboardNavigationContextHelper.js

```diff
@@ -0,0 +1,51 @@
+import { NAV_DIRECTIONS } from "../../hooks/useFullKeyboardListeners";
+
+export const getDirectionMaps = positions => {
+  const directionMaps = {
+    [NAV_DIRECTIONS.RIGHT]: new Map(),
+    [NAV_DIRECTIONS.LEFT]: new Map(),
+    [NAV_DIRECTIONS.UP]: new Map(),
+    [NAV_DIRECTIONS.DOWN]: new Map()
+  };
+  positions.forEach(position => {
+    if (position.topElement && position.bottomElement) {
+      directionMaps[NAV_DIRECTIONS.UP].set(position.bottomElement, position.topElement);
+      directionMaps[NAV_DIRECTIONS.DOWN].set(position.topElement, position.bottomElement);
+    }
+    if (position.leftElement && position.rightElement) {
+      directionMaps[NAV_DIRECTIONS.LEFT].set(position.rightElement, position.leftElement);
+      directionMaps[NAV_DIRECTIONS.RIGHT].set(position.leftElement, position.rightElement);
+    }
+  });
+  return directionMaps;
+};
+
+export const getOppositeDirection = direction => {
+  if (direction === NAV_DIRECTIONS.LEFT) return NAV_DIRECTIONS.RIGHT;
+  if (direction === NAV_DIRECTIONS.RIGHT) return NAV_DIRECTIONS.LEFT;
+  if (direction === NAV_DIRECTIONS.UP) return NAV_DIRECTIONS.DOWN;
+  if (direction === NAV_DIRECTIONS.DOWN) return NAV_DIRECTIONS.UP;
```

Comment:
We should probably throw an explicit error if passed a non-supported direction, otherwise there will be no return value, and debugging would be harder.

Also [PETTY] Can be a `switch` case.

Reply:
done 👍